hog/ImagePoint center
uint16 width
uint16 height
hog/ImagePoint[4] cornerPoints
